cmd/nginx/flags.go

```diff
@@ -201,6 +201,16 @@ func parseFlags() (bool, *controller.Configuration, error) {
 		glog.Warningf("Disabling dynamic configuration feature (LuaJIT is not available in s390x and ppc64le)")
 	}
 
+	// LuaJIT is not available on arch s390x and ppc64le
```

Comment:
this makes the logic above obsolete now, right? can you delete it?

Reply:
oh right, i will remove it